src/pymor/reductors/neural_network.py

```diff
@@ -441,8 +521,9 @@ class NeuralNetworkStatefreeOutputReductor(NeuralNetworkReductor):
         See :class:`~pymor.reductors.neural_network.NeuralNetworkReductor`.
     """
 
-    def __init__(self, fom=None, training_set=None, validation_set=None, validation_ratio=0.1,
-                 validation_loss=None, scale_inputs=True, scale_outputs=False):
+    def __init__(self, fom=None, reduced_basis=None, training_set=None, validation_set=None, training_snapshots=None,
```

Comment:
If I do not add it here it will give an error in the ```reduce``` method of the ```NeuralNetworkReductor``` when it asks whether it is None. If we do not want it here we should change that line to a ```hasattr()``` like statement.

Reply:
I am wrong. I removed it.